### DISCUSSION TITLE
How do rounds work in round-robin?

### DISCUSSION THREAD

**Discussion by IzStriker** [evroon/bracket#696] [Ideas]
~~Hi, I'm trying to workout how rounds fit into the structure of the application. So I've created a stage with round robin, but when I go to the schedule page the games are seperated into rounds. I don't quite understand their purpose, so they can be set to active, but I think I would rather just be able to set an individual game to active. Also, I don't understand how the program decides which games or the number of games to put in a given round. If any insight could be provided it would be much appreicated.~~

I understand that each round contains the max number of concurent games that can be played at once given the number of players in a round robin tournament. However, I don't think always putting th max number of concurrent games in each round makes sense. For example, what if we only have one court avaiable. I think it makes more sense to show the court schedule as an alernative view on the page where the user enters score as it's less confusing.

**Comment by evroon** [comment]
> I think it makes more sense to show the court schedule as an alernative view on the page where the user enters score as it's less confusing.

I agree, I had the same idea but there was no issue for it yet. But now I created a more actionable issue for it: https://github.com/evroon/bracket/issues/695
